docs/configuration/configuration.md

```diff
@@ -100,6 +100,7 @@ An example [oauth2-proxy.cfg]({{ site.gitweb }}/contrib/oauth2-proxy.cfg.example
 | `-request-logging-format` | string | Template for request log lines | see [Logging Configuration](#logging-configuration) |
 | `-resource` | string | The resource that is protected (Azure AD only) | |
 | `-reverse-proxy` | bool | are we running behind a reverse proxy, controls whether headers like X-Real-Ip are accepted | false |
+| `-real-client-ip-header` | string | Header used to determine the real IP of the client (one of: X-Forwarded-For, X-Real-IP, or X-ProxyUser-IP) | X-Real-IP |
```

Comment:
Might be worth noting this relies on `ReverseProxy` being set, WDYT?

Reply:
Definitely, I'll update it.

Reply:
```suggestion
| `-real-client-ip-header` | string | Header used to determine the real IP of the client, has no effect without the `-reverse-proxy` (one of: X-Forwarded-For, X-Real-IP, or X-ProxyUser-IP) | X-Real-IP |
```